### ISSUE TITLE
Undefined Index Notice - Elementor 2.81 & Pro 2.7.3

### ISSUE THREAD

**Issue by colbyalbo** [elementor/elementor#9848]
## Prerequisites

- [x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x ] The issue still exists against the latest stable version of Elementor.

## Steps to reproduce
activate Elementor and Elementor pro, enable debug log.

## Isolating the problem

- [ x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ x] This bug happens with a default WordPress theme active.
- [x
[debug.log](https://github.com/elementor/elementor/files/3953226/debug.log)
 ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by colbyalbo** [comment]
[debug.log](https://github.com/elementor/elementor/files/3953227/debug.log)

**Comment by colbyalbo** [comment]
[11-Dec-2019 23:51:42 UTC] PHP Notice:  Undefined index: before in D:\code\wf-bravo\wp-content\plugins\elementor\core\dynamic-tags\tag.php on line 41

**Comment by shilo-ey** [comment]
Duplicate of #9847

**Comment by shilo-ey** [comment]
Thanks for reporting, we will address this issue ASAP and release a patch

**Comment by colbyalbo** [comment]
> 
> 
> Thanks for reporting, we will address this issue ASAP and release a patch

Thanks!

**Comment by colbyalbo** [comment]
I can confirm that this has been fixed on my installs of v2.8.2

thanks!
